src/main/java/com/javabom/lotto/domain/shop/LottoNumbersGenerator.java

```diff
@@ -0,0 +1,31 @@
+package com.javabom.lotto.domain.shop;
+
+import com.javabom.lotto.domain.ticket.LottoNumber;
+import com.javabom.lotto.domain.ticket.LottoTicket;
+
+import java.util.*;
+import java.util.stream.Collectors;
+import java.util.stream.IntStream;
+
+public class LottoNumbersGenerator {
+
+    private final List<LottoNumber> lottoNum;
+
+    public LottoNumbersGenerator() {
+        lottoNum = IntStream.rangeClosed(LottoNumber.NUMBER_BEGIN, LottoNumber.NUMBER_END)
+                .mapToObj(LottoNumber::new)
+                .collect(Collectors.toList());
+    }
+
+    public List<LottoNumber> generate() {
+        Collections.shuffle(lottoNum);
+        return new ArrayList<LottoNumber>(lottoNum.subList(0, LottoTicket.LOTTO_NUMBER_COUNT)) {
+        };
+    }
```

Comment:
```suggestion
    public List<LottoNumber> generate() {
        Collections.shuffle(lottoNum);
        return new ArrayList<>(lottoNum.subList(0, LottoTicket.LOTTO_NUMBER_COUNT));
    }
```
<> 생략이 가능하구 {}는 필요없습니다.

Reply:
또, 랜덤 관련해서 테스트가 불가능해보입니다!